kratos/sources/model_part_io.cpp

```diff
@@ -220,6 +220,105 @@ void ModelPartIO::WriteProperties(PropertiesContainerType const& rThisProperties
     }
 }
 
+void ModelPartIO::ReadGeometry(NodesContainerType& rThisNodes, GeometryType::Pointer& pThisGeometries)
```

Comment:
```suggestion
void ModelPartIO::ReadGeometry(
    NodesContainerType& rThisNodes,
    GeometryType::Pointer& pThisGeometries)
```
(minor style guide comment)